ccmbioinfo/lfs
# CARDS-2069 - Patient portal: add support for an "Affiliation" logo

**Includes**:
* [CARDS-2069](https://phenotips.atlassian.net/browse/CARDS-2069) - _Patient portal: add support for an "Affiliation" logo_
* [CARDS-2057](https://phenotips.atlassian.net/browse/CARDS-2057) - _PREMs: UX polishing_
  * Only the addition of the UHN logo top-right of the survey screen

**Testing**:
* start in `prems`
  * notice the UHN logo opposite to the YE logo
  * notice the outer sides of the logo now aligning with the sides of the survey form
  * form the prems administration > patient access, check the "Patients can answer surveys without a token" checkbox to turn on the identification screen
  * navigate to `/Survey.html` to see the logo alignment on that screen (note the actual login form might not load due to other missing settings - that is out of scope)
* start in `proms`
  * there should be no affiliation logo, also no errors due to its absence
  * check that the narrower header still looks ok

[CARDS-2069]: https://phenotips.atlassian.net/browse/CARDS-2069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CARDS-2057]: https://phenotips.atlassian.net/browse/CARDS-2057?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
